composer.json

```diff
@@ -25,20 +25,20 @@
         "php":                      "^7.0",
         "phpspec/prophecy":         "^1.5",
         "phpspec/php-diff":         "^1.0.0",
-        "sebastian/exporter":       "^1.0 || ^2.0",
-        "symfony/console":          "^2.7 || ^3.0",
-        "symfony/event-dispatcher": "^2.7 || ^3.0",
-        "symfony/process":          "^2.7 || ^3.0",
-        "symfony/finder":           "^2.7 || ^3.0",
-        "symfony/yaml":             "^2.7 || ^3.0",
+        "sebastian/exporter":       "^2.0",
+        "symfony/console":          "^3.2",
```

Comment:
Some of them will allow code to be removed that was required to support 2.7 and 3.x – another commit has been added to this repository:

https://github.com/phpspec/phpspec/pull/1051/files#diff-bc97c89cde2c88ff4eac8d8009db7b42L86